ndio/remote/neurodata.py

```diff
@@ -208,6 +208,29 @@ def get_public_datasets_and_tokens(self):
                 datasets[dataset] = [t]
         return datasets
 
+    def getURL(self, url, token='', json=''):
```

Comment:
I'm fine with moving this to the top, in general assessor methods should be up there anyways so I'm in favor of this.